packages/monitors/src/exchange-rates-mainnet.ts

```diff
@@ -0,0 +1,79 @@
+import { ethers, FixedNumber, providers } from 'ethers';
+import { IAggregatorABI } from '@notional-finance/contracts';
+import { aggregate } from '@notional-finance/multicall';
+import { log } from '@notional-finance/logging';
+import {
+  AggregateCallList,
+  JobOptions,
+  MonitorJob,
+  OracleContract,
+} from './types';
+
+let provider: ethers.providers.JsonRpcBatchProvider;
+
+function getCalls(): AggregateCallList {
+  const contracts: OracleContract[] = [
+    ['eth_usd', '0x5f4ec3df9cbd43714fe2740f5e3616155c5b8419', 8],
```

Comment:
Can we also add another attribute to each one of these contracts? It would be either "stable" or volatile" depending on the nature of the exchange rate. All of these are "volatile" with the exception of:

wtbc_btc
usdc_usd
dai_usd

Reply:
In the log pipeline, I add the currency_type attribute, you should see usdc_usd & dai_usd currently have the attribute, I can add it to wtbc_btc.

Reply:
I feel like it is going to be mistake prone to maintain state attributes like that across two systems since it is not obvious that is happening here. I'd prefer to have all the configuration information in the code so that we can see and review it in one place.

Reply:
Added `volatilityType` configuration, updated datadog metrics and monitors.